controller/rest/system.go

```diff
@@ -330,6 +330,16 @@ func handlerSystemGetConfigBase(apiVer string, w http.ResponseWriter, r *http.Re
 						RegistryHttpsProxyEnable: rconf.RegistryHttpsProxyEnable,
 						RegistryHttpProxy:        rconf.RegistryHttpProxy,
 						RegistryHttpsProxy:       rconf.RegistryHttpsProxy,
+						RegistryHttpProxyCfg: api.RESTProxyConfig{
```

Comment:
handlerSystemGetConfigBase() is for GET("/v1/system/config") & GET("/v2/system/config").
Why add fields RegistryHttpProxyCfg & RegistryHttpsProxyCfg (both look like for PATCH purpose) in RESTSystemConfigProxyV2 which is for GET?

Reply:
yeah that's correct.  I added it to GET because it would be more consistent with REST concept, where every data is a resource.  Any concern about it?  The data in consul remains unchanged.

Reply:
For example, when I’m calling PATCH /v2/system/config, I would expect that the field I’m patching also exists in its GET response.  Hope this clarifies things a bit.

Reply:
I think this is the only type among all response types for GET requests that has this kind of fields.
So RegistryHttpProxyCfg/RegistryHttpsProxyCfg will always have the same value as RegistryHttpProxy/RegistryHttpsProxy in the response json string, correct?

Reply:
Yes, that’s correct. It’s purely for REST API.